### ISSUE TITLE
Error in tsd when I am trying to do npm install, 

### ISSUE THREAD

**Issue by ramyarao242** [DefinitelyTyped/tsd#231]
Hi,
I am stuck with this problem for over a day now, when i am trying to do npm install i am getting the following error:
[ERR!] cwd  : c:\p4\Shared\seedProjects\angular2
[ERR!] os   : Windows_NT 6.1.7601
[ERR!] argv : "node" "c:\\p4\\Shared\\seedProjects\\angular2\\node_modules\\tsd\
\build\\cli.js" "reinstall" "--overwrite"
[ERR!] node : v0.10.22
[ERR!] tsd  : 0.6.5
[ERR!] Error: tunneling socket could not be established, cause=9408:error:140770
FC:SSL routines:SSL23_GET_SERVER_HELLO:unknown protocol:openssl\ssl\s23_clnt.c:766:
[ERR!] CODE : ECONNRESET
[ERR!] tunneling socket could not be established, cause=9408:error:140770FC:SSL
routines:SSL23_GET_SERVER_HELLO:unknown protocol:openssl\ssl\s23_clnt.c:766:

This is most likely not a problem with tsd itself
and is related to network connectivity.
In most cases you are behind a proxy or have bad network settings.

If you are behind a proxy, please make sure that the
'proxy' config is set properly.  See: https://github.com/DefinitelyTyped/tsd#tsd
rc
[ERR!] .tsdrc could not be retrieved

[ERR!] Please include the following file with any support request:
    c:\p4\Shared\seedProjects\angular2\tsd-debug.log

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Programs\\nodejs\\\\node.exe" "C:\\Programs\\nodejs\\node_mod
ules\\npm\\bin\\npm-cli.js" "run" "tsd-update"
npm ERR! node v0.10.22
npm ERR! npm  v2.9.1
npm ERR! code ELIFECYCLE
npm ERR! Angular-2-Seed@0.0.0 tsd-update: `npm run remove-tsd-loader-typings &&
tsd reinstall --overwrite`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the Angular-2-Seed@0.0.0 tsd-update script 'npm run remove-ts
d-loader-typings && tsd reinstall --overwrite'.
npm ERR! This is most likely a problem with the Angular-2-Seed package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run remove-tsd-loader-typings && tsd reinstall --overwrite
npm ERR! You can get their info via:
npm ERR!     npm owner ls Angular-2-Seed
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     c:\p4\Shared\seedProjects\angular2\npm-debug.log

npm ERR! Windows_NT 6.1.7601
npm ERR! argv "C:\\Programs\\nodejs\\\\node.exe" "C:\\Programs\\nodejs\\node_mod
ules\\npm\\bin\\npm-cli.js" "install"
npm ERR! node v0.10.22
npm ERR! npm  v2.9.1
npm ERR! code ELIFECYCLE
npm ERR! Angular-2-Seed@0.0.0 postinstall: `npm run tsd-update && tsd install &&
 tsd link`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the Angular-2-Seed@0.0.0 postinstall script 'npm run tsd-upda
te && tsd install && tsd link'.
npm ERR! This is most likely a problem with the Angular-2-Seed package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     npm run tsd-update && tsd install && tsd link
npm ERR! You can get their info via:
npm ERR!     npm owner ls Angular-2-Seed
npm ERR! There is likely additional logging output above.

npm ERR! Please include the following file with any support request:
npm ERR!     c:\p4\Shared\seedProjects\angular2\npm-debug.log

What is the error, and what steps should i follow to do an npm install and then a gulp build? And also will npm-debug.log be generated once its built?

**Comment by nycdotnet** [comment]
Let's see if it's actually related to tsd.  I suspect it is your network setup.

From your command line, create a new folder.  Go in there and type `npm init`.  Press enter several times to accept all the defaults.  Then type `npm install typescript --save`.

You should get output like this:

```shell
npm WARN package.json yourfoldername@1.0.0 No description
npm WARN package.json yourfoldername@1.0.0 No repository field.
npm WARN package.json yourfoldername@1.0.0 No README data
typescript@1.6.2 node_modules\typescript
```

You should also see a `package.json` file in the folder with a dependencies section like this:
```json
"dependencies": {
    "typescript": "^1.6.2"
  }
```

Also TypeScript should be available to you.  if you run `node node_modules/typescript/lib/tsc.js --version`, you should get this output.
```shell
message TS6029: Version 1.6.2
```

Please confirm that all of the above works.  If not, there is some sort of network issue going on which is unrelated to tsd.  You could try upgrading your version of Node (you have v0.10.22 which is fairly old now) or npm (you have 2.9.1 which is from last May).

**Comment by ramyarao242** [comment]
Yes, everything you say is working... but i am still getting that error..

**Comment by flacle** [comment]
@ramyarao242 wondering if in the mean time you've found a solution to that error, I'm also having an install issue with `tsd install`.
